src/EditorFeatures/Test2/InteractivePaste/InteractivePasteCommandHandlerTests.vb

```diff
@@ -0,0 +1,139 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Text
+Imports System.Windows
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Editor.Commands
+Imports Microsoft.CodeAnalysis.Editor.Host
+Imports Microsoft.CodeAnalysis.Editor.Implementation.Interactive
+Imports Microsoft.CodeAnalysis.Editor.Shared.Extensions
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Text.Shared.Extensions
+Imports Microsoft.VisualStudio.Text
+Imports Microsoft.VisualStudio.Text.Editor
+Imports Microsoft.VisualStudio.Text.Operations
+Imports Microsoft.CodeAnalysis.Editor.CommandHandlers
+Imports Microsoft.VisualStudio.InteractiveWindow
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.InteractivePaste
+    Public Class InteractivePasteCommandhandlerTests
+        Private Function CreateCommandHandler(workspace As TestWorkspace) As InteractivePasteCommandHandler
+            Dim handler = New InteractivePasteCommandHandler(workspace.GetService(Of IEditorOperationsFactoryService),
+                                                             workspace.GetService(Of ITextUndoHistoryRegistry))
+            handler.RoslynClipBoard = New MockClipboard()
+            Return handler
+        End Function
+
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.Interactive)>
+        Public Sub PasteCommandWithInteractiveFormat()
+            Using workspace = TestWorkspaceFactory.CreateWorkspace(
+                    <Workspace>
+                        <Project Language="C#" CommonReferences="true">
+                            <Document/>
+                        </Project>
+                    </Workspace>)
+
+                ' Force initialization.
+                workspace.GetOpenDocumentIds().Select(Function(id) workspace.GetTestDocument(id).GetTextView()).ToList()
```

Comment:
I had problem with the test and though I needed this line (copied from other tests) to force initialization of `workspace`. Turns out I don't, removed.